Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/Infrastructure/TargetConstants.cs

```diff
@@ -121,8 +121,8 @@ internal static class TargetProperties
     public const string AdditionalFilesItemType = "AdditionalFiles";
 
     public const string SonarProjectOutFolderFilePath = "SonarProjectOutFolderFilePath";
-    public const string SonarProjectConfigFilePath = "SonarProjectConfigFilePath";
-    public const string ProjectSpecificOutDir = "ProjectSpecificOutDir";
+    public const string SonarProjectConfigFilePath = "SonarProjectConfigFilePath from parameter ProjectConfigFilePath"; // Issue in the MsBuild structured logger.
+    public const string ProjectSpecificOutDir = "ProjectSpecificOutDir from parameter UniquePath"; // Issue in the MsBuild structured logger.
```

Comment:
I'm having trouble understanding this change. Why isn't a path or a directory assigned to "{..}Path" and "{..}Dir" constants?

Reply:
This represents the MsBuild property name that contains the actual value. We are using MsBuild Structured Logger library to read the .binlog and read the value of the property.